panda/src/downloader/httpDate.cxx

```diff
@@ -226,7 +226,7 @@ HTTPDate(const string &format) {
     time_t now = time(nullptr);
     struct tm *tp = localtime(&now);
     _time -= tp->tm_gmtoff;
-#elif defined(_WIN32)
+#elif defined(_MSC_VER)
     long int timezone;
     _get_timezone(&timezone);
```

Comment:
This is tricky. `_get_timezone` is actually part of the Microsoft C runtime library, so protecting it with `_MSC_VER` (which checks for the compiler) isn't *quite* right.

Reply:
Allrighty, I will check if there's a macro that checks if C runtime library is available.